libs/api/domains/financial-aid/src/lib/financialAid.module.ts

```diff
@@ -0,0 +1,36 @@
+import { Module, DynamicModule } from '@nestjs/common'
+import { FinancialAidResolver } from './financialAid.resolver'
+import { FinancialAidService } from './financialAid.service'
+import { ApplicationApi, Configuration } from '../../gen/fetch'
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+
+export interface Config {
+  baseApiUrl: string
+}
+
+@Module({})
+export class FinancialAidModule {
```

Comment:
I know this is consistent with your “app” name, but I’m concerned this might conflict with some “government financial aid” programs.

Should we create a more focused domain like Veita, Samband or “LocalGov” which includes FinancialAidApplication-related resolvers and payloads?E

Reply:
Very good point, we have been using the 'Municipalities'. So going to change to `MunicipalitiesFinancialAid`, you'll let me know if that is also too vague.

Reply:
Sounds great! If anything, the domain could be “Municipalities” and contain multiple resolvers for different things. But this is fine for now.